### DISCUSSION TITLE
access to the website

### DISCUSSION THREAD

**Discussion by george1994grandmaster** [vercel/community#5135] [Help]
### Summary

Hello.  This site cannot be reached.  I get this message already already second day and i am a bit confuced what to do. Thanks in advance.

### Example

https://e-commerce-elite-accessory.vercel.app/

### Steps to Reproduce

Hello.  This site cannot be reached.  I get this message already already second day and i am a bit confuced what to do. Thanks in advance.

**Comment by amyegan** [comment]
Hi @george1994grandmaster. I'm able to visit the site without seeing any error today. Are you still having problems with it?
